core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -101,15 +104,81 @@ impl<'a> Tokenizable for CommitBatchInfo<'a> {
                 ),
                 // `systemLogs`
                 Token::Bytes(serialize_commitments(&self.0.header.system_logs)),
-                // `totalL2ToL1Pubdata`
-                Token::Bytes(
-                    self.0
-                        .header
-                        .pubdata_input
-                        .clone()
-                        .unwrap_or(self.0.construct_pubdata()),
+            ]
+        }
+    }
+}
+
+impl<'a> Tokenizable for CommitBatchInfo<'a> {
+    fn from_token(_token: Token) -> Result<Self, Web3ContractError>
+    where
+        Self: Sized,
+    {
+        // Currently there is no need to decode this struct.
+        // We still want to implement `Tokenizable` trait for it, so that *once* it's needed
+        // the implementation is provided here and not in some other inconsistent way.
+        Err(Web3ContractError::Api(Web3ApiError::Decoder(
+            "Not implemented".to_string(),
+        )))
+    }
+
+    fn into_token(self) -> Token {
```

Comment:
do we have a test that checks the output tokens ? (even a simple one that compares them with the values from file)